buildSrc/build.gradle

```diff
@@ -53,9 +53,8 @@ props.put("elasticsearch", esVersion)
 props.put("build-tools", buildToolsVersion)
 
 repositories {
-    jcenter()
```

Comment:
JCenter has been deprecated itself and is also marked as deprecated in gradle. For this build we can safely replace it with `gradlePluginPortal` here to e.g. resolve the shadow plugin